### PR TITLE
Add a condition to destroy toolbar, if it doesn't exist with 'showToo…

### PR DESCRIPTION
**Pull request by edellucien**
…lbar=false'

### CODE REVIEW

**Comment by edellucien** [comment]
Still actual for ole integration in plan/editor.

File: src/control/control.js [hunk 1]
```diff
@@ -19,8 +20,13 @@ class Control extends ol.control.Control {
...
+    let button = null;
+    if (!options.element) {
+      button = document.createElement('button');
+      button.className = `ole-control ${options.className}`;
+    } else {
+      button = options.element;
+    }
 
     super({
       element: button,
```
Comment (oterral) [inline src/control/control.js]:
```suggestion
      element: options.element || button,
```

File: src/control/control.js [hunk 2]
```diff
@@ -19,8 +20,13 @@ class Control extends ol.control.Control {
    * @param {function} [options.layerFilter] Filter editable layer.
    */
   constructor(options) {
-    const button = document.createElement('button');
-    button.className = `ole-control ${options.className}`;
+    let button = null;
+    if (!options.element) {
+      button = document.createElement('button');
+      button.className = `ole-control ${options.className}`;
+    } else {
```
Comment (oterral) [inline src/control/control.js]: you can remove the else with my suggestion

File: src/control/control.js [hunk 3]
```diff
@@ -55,11 +61,15 @@ class Control extends ol.control.Control {
      */
     this.title = options.title;
 
-    const img = document.createElement('img');
-    img.src = options.image;
+    if (!options.element) {
```
Comment (oterral) [inline src/control/control.js]:
```suggestion
    if (button) {
```

**Pull request merged**